shared/fs/popups/row-action-popup-container.js

```diff
@@ -120,6 +125,15 @@ const getRootMenuItems = (stateProps, dispatchProps) => {
       title: 'Download a copy',
       onClick: () => download(path),
     })
+  !!pathItem.tlfMeta &&
+    ['public', 'private'].includes(visibility) &&
```

Comment:
Does this mean we don't allow ignoring team TLFs and is that intentional?

Reply:
(Cecile said offline that that's intentional)

Reply:
That's right. There's no way to unignore team TLFs in the GUI, vs. public or private folders can be unignored using the widget.

Reply:
You also can't ignore `/keybase/private/you` and `/keybase/public/you`